lib/utils.js

```diff
@@ -489,6 +489,15 @@ api.verifyCapabilityChain = async ({
     if(!valid) {
       throw error;
     }
+    // FIXME: the security context defines `expires` and `expiration` as
+    // equivalent terms, how does that affect this code?
```

Comment:
Use `expires`. `expiration` is deprecated -- and any JSON-LD compaction process will choose `expires` over `expiration` due to the "shortest term wins" compaction term selection rule.